examples/kubernetes/snapshot/source_pvc.yaml

```diff
@@ -0,0 +1,11 @@
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: source-pvc
+spec:
+  accessModes:
+    - ReadWriteOnce
+  storageClassName: csi-gce-pd
+  resources:
+    requests:
+      storage: 1Gi
```

Comment:
I don't remember if `1Gi` is below the "minimum" required size. Probably not if you've tested this and it works.

Could we reuse PVC's from a different example? I guess there is value to having all the pieces of this user guide in the same folder

Reply:
Used `6Gi` now. I had `1Gi` minimum required size in my setup.